packages/api/tests/endpoints/providers.js

```diff
@@ -88,6 +90,7 @@ test('CUMULUS-911 POST without an Authorization header returns an Authorization
 
   return testEndpoint(providerEndpoint, request, (response) => {
     assertions.isAuthorizationMissingResponse(t, response);
+    t.is(JSON.parse(response.body).record, undefined);
```

Comment:
Assuming this is testing CUMULUS-917, it would be better to write a separate test that tries to create a provider without auth credendials, then list providers and make sure that the new provider is not present.

Reply:
When a provider is created it is written to DynamoDB using `packages/api/models/providers.js`.  You could use that model in your test to see if the new provider was created in DynamoDB.

Reply:
OK. I think I'll have to update these tests to include provider data in the payload so I can verify that provider data was not created, but that isn't a problem